src/main/java/seedu/revision/ui/MainWindow.java

```diff
@@ -71,6 +71,32 @@ public void handleStart(Mode mode) throws CommandException {
         }
     }
 
+    /**
+     * Shows history of results as a line graph. Loads the window components for line graph.
+     * @throws CommandException
+     */
+    @FXML
+    public void handleHistory() throws CommandException {
+        if (this.logic.getStatisticsList().size() > 0) {
+
+        } else {
+            throw new CommandException("No past results were found.");
+        }
+    }
+
+    /**
+     * Shows results of latest quiz attempted as a pie chart. Loads the window components for pie chart.
+     * @throws CommandException
+     */
+    @FXML
+    public void handleStats() throws CommandException {
+        if (this.logic.getStatisticsList().size() > 0) {
+
```

Comment:
This one also

Reply:
supposed to show pie chart, task still ongoing